BackofficeBundle/Resources/public/coffee/underscoreTemplateLoader.coffee

```diff
@@ -37,14 +37,16 @@ do ->
       if !@templates[language][templateName]
         templateLoader = this
         filename = appRouter.generateUrl('loadUnderscoreTemplate')
-        jQuery.get filename, {
-          'language': language
-          'templateId': templateName
-        }, (tpl) ->
-          templateLoader.addTemplate templateName, language, tpl
-          templateLoader.storeTemplates()
-          view.onTemplateLoaded templateName, tpl
-          return
+        jQuery.ajax {
+          method: "GET"
+          url: filename
+          data: { 'language': language,'templateId': templateName }
+          success: (tpl, textStatus, xhr) ->
```

Comment:
effectively, "success" is involved in a number of code http: (status > = 200 && status < 300 || === status 304)